jcl/j9modules.txt

```diff
@@ -44,8 +44,9 @@ java.management
 # requires java.management
 jdk.management
 
-# requires jdk.management
-com.ibm.management
+#requires jdk.management, processed to require the right module name
```

Comment:
"processed to require the right module name" doesn't seem provide more info.

Reply:
Would "processed by j9modules.xml file to require the right module name" be better?